src/Stratis.Bitcoin.Features.RPC/RpcSettings.cs

```diff
@@ -36,24 +36,16 @@ public class RpcSettings
         /// <summary>List of IP addresses that are allowed to connect to RPC interfaces.</summary>
         public List<IPAddress> AllowIp { get; set; }
 
-        private Action<RpcSettings> callback = null;
-
         /// <summary>
         /// Initializes an instance of the object.
         /// </summary>
-        public RpcSettings()
+        public RpcSettings(NodeSettings nodeSettings)
         {
             this.Bind = new List<IPEndPoint>();
             this.DefaultBindings = new List<IPEndPoint>();
             this.AllowIp = new List<IPAddress>();
-        }
 
-        /// <summary> Initializes an instance of the object.</summary>
-        /// <param name="callback">The callback to call after the settings have been loaded.</param>
-        public RpcSettings(Action<RpcSettings> callback)
-            : this()
-        {
-            this.callback = callback;
+            this.Load(nodeSettings);
```

Comment:
Can you bring the content of the Load(NodeSettings) method inside the ctor?

Reply:
Ah, missed this one, ty.
Fixed